docs/api.swagger.json

```diff
@@ -287,6 +287,13 @@
             "in": "path",
             "required": true
           },
+          {
+            "type": "string",
```

Comment:
@sum2000 the member parameter is added properly here

Reply:
@aeneasr I added these manually to test the api but automatic generation from swagger spec generate doesn't add it on my system. If you notice, the policy queries are not added. I will uninstall go-swagger and reinstall it and try again.

Reply:
Also, swagger 2.0 allows arrays of native types to be passed in query, so Member []string should work.